src/components/space/SyncAdvancedScreen.js

```diff
@@ -0,0 +1,206 @@
+import Banner from '../common/Banner';
+import Loader from '../common/Loader';
+import { clearSpacesForSync } from '../../actions';
+import './SpaceScreen.css';
+import Styles from '../../Styles';
+import { HOME_PATH } from '../../config/paths';
+import SpaceNotFound from './SpaceNotFound';
+import { SYNC_SPACE_PROPERTIES } from '../../config/constants';
+import SyncCancelButton from './sync/SyncCancelButton';
+import SyncAcceptButton from './sync/SyncAcceptButton';
+import { SYNC_ADVANCED_MAIN_ID } from '../../config/selectors';
+
+const diffEditorStyles = {
+  variables: {
+    highlightBackground: '#fefed5',
+    highlightGutterBackground: '#ffcd3c',
+  },
+  line: {
+    padding: '10px 2px',
+    wordBreak: 'break-word',
+    '&:hover': {
+      background: '#a26ea1',
+    },
+  },
+  titleBlock: {
+    pre: {
+      fontSize: '16px',
+      fontWeight: 'bold',
+      fontFamily: 'arial',
+    },
+  },
+};
+
+class SyncAdvancedScreen extends Component {
+  state = {
+    openDrawer: false,
+  };
+
+  static propTypes = {
+    localSpace: ImmutablePropTypes.contains({
+      id: PropTypes.string.isRequired,
+      description: PropTypes.string,
+      name: PropTypes.string.isRequired,
+      deleted: PropTypes.bool.isRequired,
+    }).isRequired,
+    remoteSpace: ImmutablePropTypes.contains({
+      id: PropTypes.string,
+      description: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }).isRequired,
+    dispatchClearSpaces: PropTypes.func.isRequired,
+    activity: PropTypes.bool.isRequired,
+    classes: PropTypes.shape({
+      root: PropTypes.string.isRequired,
+      appBar: PropTypes.string.isRequired,
+      appBarShift: PropTypes.string.isRequired,
+      drawer: PropTypes.string.isRequired,
+      drawerPaper: PropTypes.string.isRequired,
+      drawerHeader: PropTypes.string.isRequired,
+      content: PropTypes.string.isRequired,
+      menuButton: PropTypes.string.isRequired,
+      contentShift: PropTypes.string.isRequired,
+      hide: PropTypes.string.isRequired,
+      button: PropTypes.string.isRequired,
+    }).isRequired,
+    theme: PropTypes.shape({ direction: PropTypes.string.isRequired })
+      .isRequired,
+    history: PropTypes.shape({
+      goBack: PropTypes.func.isRequired,
+      push: PropTypes.func.isRequired,
+      replace: PropTypes.func.isRequired,
+    }).isRequired,
+    t: PropTypes.func.isRequired,
+  };
+
+  componentDidUpdate() {
+    const {
+      localSpace: { deleted },
+      history: { replace },
+    } = this.props;
+    // redirect to home if space is deleted
+    if (deleted) {
+      replace(HOME_PATH);
+    }
+  }
+
+  componentWillUnmount() {
+    const { dispatchClearSpaces } = this.props;
+    dispatchClearSpaces();
+  }
+
+  render() {
+    const { localSpace, remoteSpace, activity, classes, t } = this.props;
+    const { name } = localSpace;
+    const { openDrawer } = this.state;
+
+    const filteredSpace = _.pick(localSpace, SYNC_SPACE_PROPERTIES);
```

Comment:
Add comment here explaining that you only show a selection of the space properties. In general, add more comments too, for ease of understanding this functionality.